spinn_front_end_common/interface/interface_functions/graph_data_specification_writer.py

```diff
@@ -162,10 +162,18 @@ def __generate_data_spec_for_vertices(
                         FecDataView.get_max_run_time_steps())
                     total_est_size += est_size
                     if size > est_size:
-                        logger.warning(
-                            "Region {} of vertex {} is bigger than "
-                            "expected: {} estimated vs. {} actual",
-                            i, vertex.label, est_size, size)
+                        raise ValueError(
```

Comment:
if intended here approved!

Reply:
Yes, I think this is worth having; makes absolutely sure!